js/cm-api-v2/GetVariantsWithComponentsOfType.js

```diff
@@ -0,0 +1,21 @@
+// DocSection: cm_api_v2_get_variants_with_components_of_type
```

Comment:
The proposed code sample ID is too long. Also, this endpoint already has a REST code sample with the ID `cm_api_v2_get_variants_with_components_of_type`. Please use that one.